Build/Common.Build.props

```diff
@@ -86,6 +86,8 @@
         %(PreprocessorDefinitions);
         BYTECODE_TESTING=1
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(BuildWabt)'=='true'">%(PreprocessorDefinitions);EXCLUDE_WABT=0</PreprocessorDefinitions>
```

Comment:
nit:  May want to use a positive condition like ENABLE_WABT, instead of a negative one like EXCLUE_WABT to be slightly clearer?